src/components/Navigation.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { NavigationLink } from './NavigationLink';
+
+export const Navigation: React.FC = () => {
```

Comment:
you can return right away in arrow functions if you don't have any code before the return statement